alibi/tests/utils.py

```diff
@@ -90,3 +101,34 @@ def issorted(arr, reverse=False):
         return np.all(arr[::-1][:-1] <= arr[::-1][1:])
 
     return np.all(arr[:-1] <= arr[1:])
+
+
+@contextmanager
+def not_raises(ExpectedException):
```

Comment:
It used to be when the code was throwing errors instead of warnings. I since improved both the code and the testing code so this is just a context manager you can use to check (conditionally) that your code does not throw some errors. `no_errors` fixture would not work in that case because it will probably record some errors were thrown on some of the code pathways (so it would probs record stuff thrown in `with pytest.raises` blocks.